### ISSUE TITLE
Feature Request: Markdown Support

### ISSUE THREAD

**Issue by sunjam** [PhieF/CarnetNextcloud#15]
I'd love Carnet to be able to edit and support my existing Markdown notes, which is what [Nextcloud Notes](https://github.com/nextcloud/notes/) app does.  It would be great if they could continue to live in their current directory rather than being moved.  Thanks for reading.

**Comment by devplan44** [comment]
Yes, that would be really useful, either GFM or Markdown Extra and saving the note in HTML

**Comment by natalie-foust** [comment]
Just want to say, if you do add this, I highly recommend the "Seamless Live Preview" format used by [typora](https://typora.io). That feature makes Typora literally my favorite app in the world and carnet would definitely take its place if it could have both that and web syncing

**Comment by johenning** [comment]
I think this feature just got a lot easier to implement: [Nextcloud Text](https://nextcloud.com/blog/nextcloud-introduces-collaborative-rich-text-editor/) announced today with Nextcloud 17.

Am I right to assume that a lot of the functionality could be reused in Carnet?

**Comment by PhieF** [comment]
Not really I'm afraid, I will have a look but Carnet doesn't work like nextcloud at all

**Comment by johenning** [comment]
Alright, I have no idea of the internal architecture. I just thought that with [Text](https://github.com/nextcloud/text) becomming the default editor and it being able edit and display .md files, there might be a way to use that functionality from Carnet.

But I see that Carnet has its own file format, so that still might be a lot of work. Maybe less UI work though?

**Comment by PhieF** [comment]
For now my main purpose is to get rid of the .sqd format, then I will be able to put notes as .md
